src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -230,13 +230,14 @@ public static string SelectProductNameForDirectory(Platform.XDG_Type dirpath)
         {
             //TODO: XDG_DATA_DIRS implementation as per GitHub issue #1060
 
+            string defaultHomePath = System.Environment.GetEnvironmentVariable("HOME") ?? Directory.GetCurrentDirectory();
```

Comment:
If you check ps-ef|grep omi, the omiserver shows root user, but "systemctl start omid"'s omiserver process is very different with "omiserver -d" run as root user directly, the systemctl mode will not have permission to access Environment variables, and I know the current directory must have permission for current special limited permission root user.